libmcount/dynamic.c

```diff
@@ -514,14 +514,33 @@ static void finish_dynamic_update()
 }
 
 /* do not use floating-point in libmcount */
-static int calc_percent(int n, int total, int *rem)
+static int calc_percent(unsigned int n, unsigned int total, unsigned int *rem)
 {
 	int quot = 100 * n / total;
 
 	*rem = (100 * n - quot * total) * 100 / total;
 	return quot;
 }
 
+void print_dynamic_status(const char *filename)
+{
+	unsigned int r = 0, q = 0;
+
+	if (stats.total) {
```

Comment:
Actually it's not a mistake and I thought it's not necessary.

Reply:
if user use verbose option that mean user expect statistic also. aren't you?

Reply:
@namhyung I think all your point has been applied except this one. how do you think so?

Reply:
I still see many review comments above not answered yet.